app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
Yeah, I just had a dig, and now I can’t see any way it could’ve been producing the behaviour I thought I’d observed. Maybe it was because it accepted either URL?